scripts/scripts.js

```diff
@@ -626,6 +626,7 @@ async function loadJQueryDateRangePicker() {
 async function loadLazy(doc) {
   const main = doc.querySelector('main');
   await loadBlocks(main);
+  await loadScript('/scripts/one-trust-geo-script.js');
```

Comment:
I would put this in delayed.js instead of scripts.js along with all the other analytics

Reply:
@amol-anand Code updated, Please review once, Thank you!